box-browse-sdk/src/main/java/com/box/androidsdk/browse/uidata/ThumbnailManager.java

```diff
@@ -150,192 +227,23 @@ private int getDefaultIconResource(final BoxItem boxItem) {
             BoxFolder boxFolder = (BoxFolder) boxItem;
             if (boxFolder.getHasCollaborations() == Boolean.TRUE) {
                 if (boxFolder.getIsExternallyOwned() == Boolean.TRUE) {
-                    return R.drawable.ic_box_browsesdk_folder_external;
+                    return R.drawable.ic_box_browsesdk_locked;
                 }
                 return R.drawable.ic_box_browsesdk_folder_shared;
             }
-            return R.drawable.ic_box_browsesdk_folder;
+            return R.drawable.ic_box_browsesdk_personal;
```

Comment:
Can we be more descriptive with the icon names? locked -> folder_external and personal -> folder_personal, linked-> weblink?